metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -716,6 +722,47 @@ class MetalsLanguageServer(
     onChange(paths).asJava
   }
 
+  private def insertPackage(path: AbsolutePath): Unit = {
```

Comment:
We could move this to a PackageProvider class, it could be used in more places and provide us with result being the needed Edit.

It would be very useful for Completions too.